src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const fixedRobot = {};
+
+  for (const key in robot) {
+    fixedRobot[robot[key]] = key;
```

Comment:
Better check in for cycle is `key` already exist in `fixedRobot` and return `null` for non unique. This way you could rewrite 33-36 lines too. And it will end program with `null` quicker without unnecessary operations.

Reply:
thanks for your advice, it really reduced the amount of code made it cleaner 👍